src/btree/bt_random.c

```diff
@@ -186,10 +187,12 @@ __wt_random_descent(WT_SESSION_IMPL *session, WT_REF **refp, bool eviction)
 
 	/* Eviction should not be tapped to do eviction. */
 	if (eviction)
-		flags = WT_READ_CACHE | WT_READ_NO_EVICT | WT_READ_NO_GEN |
-		    WT_READ_NO_WAIT | WT_READ_NOTFOUND_OK | WT_READ_RESTART_OK;
+		flags = additional_flags |
```

Comment:
I think I may have taken the style rule of "successive wrapped lines should be longer than the above line" too far.